qiskit/aqua/algorithms/minimum_eigen_solvers/qpe.py

```diff
@@ -89,16 +87,9 @@ def __init__(self,
         validate_in_set('expansion_mode', expansion_mode, {'trotter', 'suzuki'})
         validate_min('expansion_order', expansion_order, 1)
         super().__init__(quantum_instance)
-        self._state_in = state_in
 
-        if isinstance(iqft, IQFT):
-            warnings.warn('The qiskit.aqua.components.iqfts.IQFT module is deprecated as of 0.7.0 '
-                          'and will be removed no earlier than 3 months after the release. '
-                          'You should pass a QuantumCircuit instead, see '
-                          'qiskit.circuit.library.QFT and the .inverse() method.',
-                          DeprecationWarning, stacklevel=2)
+        self._state_in = state_in
         self._iqft = iqft
```

Comment:
I don't think QPE will handle a None for iqft well. The PhaseEstimationCircuit seems to need it. If its None here can we default it to an appropriate instance - yes, it was like this before but maybe this can be improved prior to the re-write

Reply:
I added that case in the `PhaseEstimationCircuit` 👍